Makefile

```diff
@@ -52,6 +55,13 @@ container-image: pre-build
 	@./hack/goreleaser.sh build
 	@echo "Docker image build successfully"
 
+# Build image for E2E tests
+container-image-e2e-test:
```

Comment:
I switched to use Goreleaser for producing test image in the follow-up PR: https://github.com/infracloudio/botkube/pull/624. So you can basically ignore that part when reviewing as this is something temporary, for local dev purposes 🙂